Vagrantfile

```diff
@@ -28,6 +28,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vm.provision :hostmanager
   end
 
+  # Ensure role dependencies are in place
+  if ['up', 'provision'].include?(ARGV.first) && !(File.directory?('roles/Stouts.mongodb'))
```

Comment:
Yeah, don't like it either. For now it's simple workaround to see how 3rd party `mongo` role works.

* One of the solutions is using 3rd party Vagrant plugin [vagrant-host-shell plugin](https://github.com/phinze/vagrant-host-shell)
As a result installing Ansible Galaxy dependencies will be part of provision, which is more clear (`File.directory` will be removed too).
* Another solution is to create another Ansible playbook to install dependencies.

It's pretty strange that Ansible can't download dependencies automatically.
BTW, good point, thanks for taking a look :+1: